lib/embed/viewer/was_seed.rb

```diff
@@ -19,6 +33,13 @@ def self.supported_types
         [:"webarchive-seed"]
       end
       
+      def thumbs_list
+        Embed::WasSeedThumbs.new(@purl_object.druid).get_thumbs_list
+      end
+      
+      def format_memento_datetime(memento_datetime)
+        DateTime.strptime(memento_datetime,"%Y%m%e%H%M%S").strftime("%b %e, %Y")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.